src/main/java/com/wego/dao/models/CachedResponse.java

```diff
@@ -0,0 +1,170 @@
+package com.wego.dao.models;
+
+import com.ning.http.client.FluentCaseInsensitiveStringsMap;
+import com.ning.http.client.Response;
+import com.ning.http.client.cookie.Cookie;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.nio.ByteBuffer;
+import java.util.List;
+import org.apache.commons.collections4.MapUtils;
+import org.apache.commons.lang3.StringUtils;
+
+public class CachedResponse implements Response {
+  private String id;
+  private String responseBody;
+  private List<Cookie> cookies;
+  private FluentCaseInsensitiveStringsMap headers;
+  private Integer statusCode;
+  private String statusText;
+
+  public int getStatusCode() {
+    return statusCode != null ? statusCode : 0;
+  }
+
+  public String getStatusText() {
+    return statusText;
+  }
+
+  public byte[] getResponseBodyAsBytes() throws IOException {
+    return new byte[0];
+  }
+
+  public ByteBuffer getResponseBodyAsByteBuffer() throws IOException {
+    return null;
+  }
+
+  public InputStream getResponseBodyAsStream() throws IOException {
+    return new ByteArrayInputStream(responseBody.getBytes());
+  }
+
+  public String getResponseBodyExcerpt(int maxLength, String charset) throws IOException {
+    return null;
```

Comment:
this is still unimplemented yet, we will not implement all override functions now. We only implement the function we need for now.